guides/common/modules/snip_supported-client-architectures.adoc

```diff
@@ -0,0 +1,15 @@
+ifdef::satellite[]
+* AMD and Intel 64-bit architectures
+* The 64-bit ARM architecture
+* IBM Power Systems, Little Endian
+* 64-bit IBM Z architectures
+endif::[]
+ifdef::orcharhino[]
+* AMD and Intel 64-bit architectures are supported for all operating systems
+* The 64-bit ARM architecture and IBM Power Systems, Little Endian, are supported for certain operating systems
++
+For more information, see {atix_service_portal_clients_url}[{project-client-name}] in the _ATIX Service Portal_.
+endif::[]
+ifndef::orcharhino,satellite[]
```

Comment:
I realize you're just moving things around and changing from codes to labels, but I'd expect that upstream is a superset of downstream. AFAIK within Satellite we don't ship any additional code to support architectures so I'd be tempted to combine satellite and upstream so you'd have `ifdef::orcharhino[]` and `ifndef::orcharhino[]`.

Reply:
I beg to differ. Satellite ships the Satellite Client repository for all those architectures.

Reply:
Oh yes, good point. We should really resolve that. Now that use COPR to build our packages we should be able to.